data/coalition/coalition missions.txt

```diff
@@ -143,3 +143,842 @@ mission "Coalition License Patch"
+			`	In the message, they apologize for taking so long to pay you, and go on to tell you about how they've been doing there. Aunaris has been working in a school there, while Inturi has been helping other residents set up their own "ship-houses", as he calls them. Apparently it's become somewhat popular there, after people saw their home was a spaceship.`
+			`	They thank you again, and say you're welcome to visit anytime.`
+
+
+
+mission "Saryd Couple Follow-Up B"
+	invisible
+	landing
+	to offer
+		has "event: saryd couple follow-up"
+		has "refused saryd couple payment"
+	source
+		government "Coalition" "Heliarch"
+	on offer
+		conversation
+			`When you land, see you've received a message. It's from Inturi and Aunaris, the Saryd couple who you helped move out to their very own home.`
+			`	Along with the message, they've sent a picture of them both tending to some flowers and short trees in a little garden they've started up near the ship. In the message they go on to tell you about how they've been doing there. Aunaris has been working in a school there, while Inturi has been helping other residents set up their own "ship-houses", as he calls them. Apparently it's become somewhat popular there, after people saw their home was a spaceship.`
+			`	They thank you again, and say you're welcome to visit anytime.`
+
+
+
+mission "Longcow Vaccines 1"
+	minor
+	name "Longcow Vaccines"
+	description "Arachi ranchers have asked you to pick up the latest vaccines for longcows on <destination>."
+	to offer
+		has "Coalition: First Contact: done"
+	source
+		attributes arach
+	destination "New Finding"
+	on offer
+		conversation
+			`You see some Arachi talking among themselves in the spaceport, occasionally glancing at and pointing at you.`
+			`	One of them finally comes to you, and says, "Greetings, Captain. Interested in performing a cargo mission for us, are you?"`
+			`	She tells you that they are a cooperative of Arachi ranchers, explains that the safety and health regulations for their business, longcow ranching, were recently updated, and that they need someone to pick up the newest vaccines for the cattle on <destination>.`
+			`	"A Jump Drive you have, so get the vaccines faster - and allow us to resume business faster - you would."`
+			choice
+				`	"You've found the perfect person for the job. I'm in."`
+				`	"I suppose that's true, but I'm not looking for courier jobs, sorry."`
+					decline
+			`	The other Arachi come over after she signals to them that you've accepted to help, and they come thank you. They say that after you've picked up the vaccines on <planet> you'll have to bring them to some planets, and wish you a good trip.`
+				accept
+
+
+
+mission "Longcow Vaccines 2"
+	name "Longcow Vaccines"
+	description "Take the <cargo> to <destination>, so that the Arachi ranchers can vaccinate their cattle of longcows."
```

Comment:
```suggestion
	description "Take the <cargo> to <destination> so that the Arachi ranchers can vaccinate their cattle of longcows."
```
No comma b/c dependent clause follows independent.